polaris.shopify.com/content/content/actionable-language.md

```diff
@@ -586,18 +586,21 @@ Use the adjective “Done” for datepickers.
 
 ---
 
-## OK vs. accept
+## Close vs. accept
 
-Use the adjective “OK” when merchants need to confirm they’ve read something, but aren’t required to legally accept terms of service before continuing. For example, use “OK” when presenting a security notification in a modal or sheet.
+Use the verb “Close” when merchants need to confirm they’ve read something, but aren’t required to legally accept terms of service before continuing. For example, use “Close” when presenting a security notification in a modal or sheet.
+
+Don't use "OK". "OK" is an exclamation, not an action. When merchants click the "Close" button, they’re not saying “OK”, they’re doing a specific action.
 
 <!-- dodont -->
 
 #### Do
 
-![modal with okay button spelled with capitalized letter O and K, no punctuation](/images/content/actionable-language/ok@2x.png)
+![modal with Close button](/images/content/actionable-language/modal-with-close@2x.png)
```

Comment:
I've added an updated image here, but it's not full width, for some reason. Still trying to sort that out. Will update! 
<img width="1497" alt="Screenshot 2023-02-16 at 5 49 48 PM" src="https://user-images.githubusercontent.com/77791660/219516103-2a1ce32e-38b0-43c9-b43b-3c8dcc0dab0e.png">